terraform/lc.tf

```diff
@@ -3,12 +3,14 @@ locals {
   staging_image_id  = "ami-0dac25c567aa57e61"
   beta_image_id     = "ami-0dac25c567aa57e61"
   gpu_image_id      = "ami-05e1d61b3348d349f"
+  aarch64prod_image_id = "ami-0575a3cc115c9f3ac"
```

Comment:
Can you add the packer that makes this to the Makefile etc too please?

Reply:
I added that to main already with https://github.com/compiler-explorer/infra/commit/1e6e8473e9c714c4b2e1b8386727bb52d85bfd93 - please do take a look at it and see if I did something daft

Reply:
nice! thanks